src/main/java/lotto/domain/Winning.java

```diff
@@ -0,0 +1,56 @@
+package lotto.domain;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public enum Winning {
+    FIRST_PLACE(6, 2000000000),
```

Comment:
```suggestion
    FIRST_PLACE(6, 2_000_000_000),
```
돈과 같이 1000단위로 구분은 다음과 같이 underbar(_)를 활용해 구현하면 코드 가독성이 좋아집니다.

Reply:
자바 7에서 등장했군요🤭
숫자가 커서 불편했는데 반영하면 더 편해지겠어요:)